tests/test_hf_api.py

```diff
@@ -901,10 +905,11 @@ def setUpClass(cls):
 
     def setUp(self):
         self.REPO_NAME_LARGE_FILE = repo_name_large_file()
-        try:
+        if os.path.exists(WORKING_REPO_DIR):
             shutil.rmtree(WORKING_REPO_DIR, onerror=set_write_permission_and_retry)
-        except FileNotFoundError:
-            pass
+        logger.info(
+            f"Does {WORKING_REPO_DIR} exist: {os.path.exists(WORKING_REPO_DIR)}"
+        )
```

Comment:
I fail to see why that would result in an improvement, but glad it does! Let's monitor the next pushes on `main` to ensure it works correctly.